cypress/tests/ui/auth.spec.ts

```diff
@@ -1,23 +1,99 @@
 describe("User Sign-up and Login", function () {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/")
```

Comment:
You could navigate to localhost without explicit url, instead you could do `cy.visit("/")`